sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -290,3 +290,24 @@ ALTER TABLE `insurance_companies` CHANGE `inactive` `inactive` TINYINT(1) NOT NU
 UPDATE `layout_options` SET `fld_length` = '8' WHERE `layout_options`.`form_id` = 'DEM' AND `layout_options`.`field_id` = 'postal_code';
 #EndIf
```

Comment:
@sjpadgett , There are some discrepancies here between the upgrade and database.sql. I plan to fix these so you can keep focusing on cqm :)